client-v2/src/actions/Fronts.js

```diff
@@ -31,6 +33,15 @@ function publishCollectionSuccess(collectionId: string): Action {
   };
 }
 
+function recordStaleFronts(frontId: string, frontIsStale: boolean): Action {
+  return {
+    type: 'RECORD_STALE_FRONTS',
+    payload: {
+      [frontId]: frontIsStale
```

Comment:
Minor nitpick I feel like `frontId` should be a property on the payload rather than just the key of the object as if we wanted to use this value in a different way in the reduce in future we'd have to use `Object.keys()`.